app-emulation/genymotion-bin/genymotion-bin-3.3.2.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 desktop pax-utils
+
+MY_PN="${PN/-bin}"
+MY_P="${MY_PN}-${PV}"
+BIN_ARCHIVE="${MY_P}-linux_x64.bin"
+
+DESCRIPTION="Complete set of tools that provide a virtual environment for Android"
+HOMEPAGE="https://genymotion.com"
+SRC_URI="${BIN_ARCHIVE}"
+
+LICENSE="genymotion"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+RDEPEND="app-emulation/virtualbox
+	|| (
+		dev-libs/openssl-compat:1.0.0
+		=dev-libs/openssl-1.0*:0
+	)
+	=dev-libs/hiredis-1.0*
+	sys-apps/util-linux
+	virtual/opengl
+"
+
+RESTRICT="bindist fetch"
+S="${WORKDIR}"
+
+QA_PREBUILT="
+	opt/${MY_PN}/*.so*
+	opt/${MY_PN}/imageformats/*.so
+	opt/${MY_PN}/plugins/*.so*
+	opt/${MY_PN}/${MY_PN}
+	opt/${MY_PN}/genyshell
+	opt/${MY_PN}/player
+	opt/${MY_PN}/${MY_PN}adbtunneld
+	opt/${MY_PN}/gmtool
+	opt/${MY_PN}/tools/*
+"
+
+pkg_nofetch() {
+	einfo
+	einfo "Please visit"
+	einfo
+	einfo "  https://www.genymotion.com/download/"
+	einfo
+	einfo "and download "
+	einfo
+	einfo "  ${BIN_ARCHIVE}"
+	einfo
+	einfo "which must be placed in DISTDIR directory."
+	einfo
+}
+
+src_unpack() {
+	cp "${DISTDIR}/${BIN_ARCHIVE}" "${WORKDIR}" || die "cp failed"
+}
+
+src_prepare() {
+	default
+
+	chmod +x ${BIN_ARCHIVE} || die "chmod failed"
+	yes | ./${BIN_ARCHIVE} > /dev/null || die "unpack failed"
+
+	# removed windows line for bashcompletion
+	sed -i "/complete -F _gmtool gmtool.exe/d" "${MY_PN}/completion/bash/gmtool.bash" || die "sed failed"
+
+	# copy .desktop file in S directory
+	sed -i -e "s:Icon.*:Icon=/opt/${MY_PN}/icons/genymotion-logo.png:" \
+		-e "s:Exec.*:Exec=/opt/${MY_PN}/genymotion:" \
+		"${HOME}"/.local/share/applications/genymobile-genymotion.desktop || die "sed failed"
+	cp "${HOME}"/.local/share/applications/genymobile-genymotion.desktop "${S}" || die "copy .desktop file"
```

Comment:
```
 * sed: can't read /var/tmp/portage/app-emulation/genymotion-bin-3.3.2/homedir/.local/share/applications/genymobile-genymotion.desktop: No such file or directory
 * ERROR: app-emulation/genymotion-bin-3.3.2::gentoo failed (prepare phase):
 *   sed failed
 * 
 * Call stack:
 *     ebuild.sh, line 136:  Called src_prepare
 *   environment, line 1766:  Called die
 * The specific snippet of code:
 *       sed -i -e "s:Icon.*:Icon=/opt/${MY_PN}/icons/genymotion-logo.png:" -e "s:Exec.*:Exec=/opt/${MY_PN}/genymotion:" "${HOME}"/.local/share/applications/genymobile-genymotion.desktop || die "sed failed";
```
I don't understand how the file ends up in portage's `${HOME}` anyway, something weird goes on with that unpacking. 

Although this error most likely comes from missing xdg package, like xdg-utils. So BDEPEND on x11-misc/xdg-utils (yes, this would never happen in a desktop system)